provider/script.go

```diff
@@ -41,18 +46,26 @@ func NewScriptProviderFromConfig(other map[string]interface{}) (IntProvider, err
 		return nil, err
 	}
 
-	return NewScriptProvider(cc.Cmd, cc.Timeout, cc.Cache)
+	return NewScriptProvider(cc.Cmd, cc.Timeout, cc.Cache, cc.Jq)
```

Comment:
cache bitte immer zuletzt

Reply:
mit "zuletzt" nehme ich an meinst Du die Reihenfolge der Parameter? Denn das speichern im Cache geschieht in Zeilte 67 ja bereits so ziemlich zuletzt.